README.md

```diff
@@ -10,15 +10,33 @@ TaskCluster Worker
 A generic worker for task cluster, written in go.
 
 
-** Warning very EARLY notes**
+Hacking
+=======
 
-Place we outline interfaces and patterns for next generation worker.
+How to get started
+------------------
+
+1. Install a version of golang > 1.4: [Download Go](https://golang.org/dl/) and set up your environment. 
+2. Get the taskcluster-worker source.  (Optionally: fork the code first, and then substitute your user name in for taskcluster below)
```

Comment:
Due to explicit package naming, the workflow would need to be:
* Fork
* Go get from taskcluster repo (so keep line 23 the same)
* Go to the checkout directory under `taskcluster` directory, add a separate remote for your fork (i.e. don't go get from your local fork)
* Work, and when you push, push to your fork (the new remote you added)

This is unfortunately the best workflow to use, since if you fork and `go get` your fork, the go imports will import the wrong version.